backend/src/appointment/routes/waiting_list.py

```diff
@@ -136,7 +136,8 @@ def invite_waiting_list_users(
 
     for id in data.id_list:
         # Look the user up!
-        waiting_list_user: models.WaitingList|None = db.query(models.WaitingList).filter(models.WaitingList.id == id).first()
+        waiting_list_user: models.WaitingList|None = \
```

Comment:
I think we can just wrap this with parenthesis instead of using \.

Reply:
(sneaked in here and edited Mels comment accordingliy 😇 )

Reply:
Oh wait I think you meant instead of using `\` ok so maybe this:

```
        waiting_list_user: models.WaitingList|None = (
            db.query(models.WaitingList).filter(models.WaitingList.id == id).first()
        )
```

Reply:
haha thank you both, updated